### DISCUSSION TITLE
Why is your CLA not the same as the .NET Foundation?

### DISCUSSION THREAD

**Discussion by jwdonahue** [stride3d/stride#1873] [Contributors]
I read your CLA, and was immediately turned off by it. While I am not a fan of complex CLA's, I at least understand the .NET Foundation's [CLA language](https://dotnetfoundation.org/docs/default-source/default-document-library/contribution-license-agreement.pdf?sfvrsn=40626e42_3).  I don't understand why you haven't adopted the .NET Foundations CLA or at least one of the other more common CLA's (the [ASF's ACL](https://apache.org/licenses/icla.pdf) also comes to mind).

I am not a lawyer, and I shouldn't need one in order to contribute to an OSS code base.  Please consider adopting one of these:

- https://apache.org/licenses/icla.pdf
- https://dotnetfoundation.org/docs/default-source/default-document-library/contribution-license-agreement.pdf?sfvrsn=40626e42_3
- https://cla.developers.google.com/about/google-individual

**Comment by manio143** [comment]
@xen2 I believe @jwdonahue is referring to https://github.com/stride3d/stride/blob/master/docs/ContributorLicenseAgreement.md - is that CLA still correct? I haven't seen the CLA bot on recent PRs but looking at one the @dotnet-policy-service has commented on it's using the [.NET Foundation CLA](https://dotnetfoundation.org/docs/default-source/default-document-library/contribution-license-agreement.pdf?sfvrsn=40626e42_3) as mentioned above.

@jwdonahue It may be that we have forgotten to remove the old CLA when moving the project under .NET Foundation. Thank you for bringing this topic up.

**Comment by xen2** [comment]
Good point, https://github.com/stride3d/stride/blob/master/docs/ContributorLicenseAgreement.md doesn't apply anymore, it was used by our own CLA bot until we started to use .NET Foundation one.

**Reply by Aggror** [reply]
I am moving the Docs away from the Stride repo and to the official documentation. Can I just remove this file and textual references to it? 
I will refer to the actual CLA that applies here:
https://dotnetfoundation.org/docs/default-source/default-document-library/contribution-license-agreement.pdf?sfvrsn=40626e42_3
